frontend/src/components/CurrentPrice.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Center, Divider, HStack, Skeleton, Text } from "@chakra-ui/react";
+import React from "react";
+import TimeStamp from "./TimeStamp";
+import { PriceInfo } from "./Types";
+
+interface PriceProps {
+    priceInfo: PriceInfo | null;
+}
```

Comment:
Would just call this `Props`. If we stick to one component per file we don't need to qualify.